### PR TITLE
Introduce rebar3_hank

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**

### CODE REVIEW

**Comment by g-andrade** [comment]
Missing CHANGELOG update, LGTM otherwise

**Review by g-andrade — APPROVED** [review]

**Comment by g-andrade** [comment]
(I can't revert my approval)

**Review by g-andrade — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,10 +14,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - facilities for easier Hex.pm publishing [Paulo Oliveira]
 - a type for decode errors [Guilherme Andrade]
 - invocation of EUnit tests under `make test` [Guilherme Andrade]
+- `rebar3_hank [Paulo Oliveira]
 
 ### Changed
 
 - library name from `cbor` to `erl_cbor` [Guilherme Andrade]
+- CI container approach to `setup-beam` with cache [Paulo Oliveira]
```
Comment (g-andrade) [inline CHANGELOG.md]: ^ Please place your new entries under a new `[Unreleased]` section and re-title the existing one to `2.0.0`, dated June 16th - the changes in question have been released. 

My brain must have not been fully functional on that day... sorry about that.

Comment (g-andrade) [reply CHANGELOG.md]: Never mind - I've fixed in in #9 . You'll just need to rebase.

**Review by g-andrade — CHANGES_REQUESTED** [review]
Please rebase in order to add the new CHANGELOG entries in the right place, following #9 (sorry about that)

**Comment by paulo-ferraz-oliveira** [comment]
Shiiiiieeeeeeeet.

**Comment by g-andrade** [comment]
image

File: CHANGELOG.md (hunk 1 above)
Comment (g-andrade) [reply CHANGELOG.md]: (I see you rebased, but the new entries are still in the wrong place)

Comment (g-andrade) [reply CHANGELOG.md]: [I guess I should have detailed "just need to rebase" better]

Comment (paulo-ferraz-oliveira) [reply CHANGELOG.md]: There's this thing here: Allow edits and access to secrets by maintainers. I left it checked.

Comment (g-andrade) [reply CHANGELOG.md]: Ahh, I considered it but thought you didn't like to use it. I'll fix the issue then

Comment (g-andrade) [reply CHANGELOG.md]: 276fd63

**Review by g-andrade — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply CHANGELOG.md]: You are half right; I don't like to use it when it's code in a major way. If it's just a small tweak, you can either propose a suggestion or change yourself, but I don't expect people to know this is how I think about the subject. 😄

**Pull request merged**